pkg/ledger/ledger_monitor.go

```diff
@@ -95,14 +95,14 @@ func (m *Monitor) Start(ctx context.Context) {
 					}
 					health = pointer.ToBool(false)
 				}
-				switch {
-				case health == nil:
-					stats.Set("ledger-health", 1, stats.T("status", "unknown"))
-				case *health == false:
-					stats.Set("ledger-health", 1, stats.T("status", "unhealthy"))
-				case *health == true:
-					stats.Set("ledger-health", 1, stats.T("status", "healthy"))
-				}
+			}
+			switch {
+			case health == nil:
```

Comment:
I don't think `health` will change from nil because the logic to mutate it is in the switch statement above guarded by  `if !m.cfg.DisableECSBehavior {...}`